package.json

```diff
@@ -104,6 +104,7 @@
     "transform": {
       "^.+\\.tsx?$": "ts-jest"
     },
+    "testEnvironment": "node",
```

Comment:
axios was using `XMLhttpRequest` because of the `jsdom` environment so this should force axios to use node's `http`.